src/parse.c

```diff
@@ -910,8 +912,9 @@ static int hyper_parse_routes(struct hyper_route **routes, uint32_t *r_num, char
 				rt->device = (json_token_str(json, &toks[++i]));
 				fprintf(stdout, "route %d device is %s\n", j, rt->device);
 			} else {
-				fprintf(stderr, "get unknown section %s in routes\n",
-					json_token_str(json, &toks[i]));
+				char *unknown = json_token_str(json, &toks[i]);
```

Comment:
there are six of these. Maybe add a helper function to reduce code duplications?